### PR TITLE
[WFLY-11876]: Upgrade Artemis to 2.7.0

### PR DESCRIPTION
**Pull request by ehsavoie**
* Updating to Apache Artemis 2.8.0
* Taking new Artemis native packaging into account
* Taking advantage of ARTEMIS-2163.
* Aligning qpid j-proton version with Artemis

Jira: https://issues.jboss.org/browse/WFLY-11876
        https://issues.jboss.org/browse/WFLY-11727

### CODE REVIEW

**Comment by ehsavoie** [comment]
Waiting for testing results

**Comment by darranl** [comment]
Are there any release notes for the changes included in this version of Artemis compared to the prior version?

**Comment by ehsavoie** [comment]
https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12342977&styleName=Html&projectId=12315920&Create=Create&atl_token=A5KQ-2QAV-T4JA-FDED%7Caca26efea9a8c56184fe65141c0d1f1bf396e637%7Clin

**Comment by ehsavoie** [comment]
FYI: there have been a 2.8.0 release to which we should upgrade https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12345169&styleName=Html&projectId=12315920&Create=Create&atl_token=A5KQ-2QAV-T4JA-FDED%7Caca26efea9a8c56184fe65141c0d1f1bf396e637%7Clin

**Review by bstansberry — APPROVED** [review]

**Review by bstansberry — COMMENTED** [review]
Removing my approval which was just a test

**Comment by ehsavoie** [comment]
The change is 'just' an artifact moved to a new GAV.

File: feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/main/module.xml [hunk 1]
```diff
@@ -47,6 +47,7 @@
         <module name="org.apache.activemq.artemis.journal" export="true"/>
         <module name="org.jboss.jts"/>
         <module name="org.jboss.logging"/>
+        <module name="org.jboss.logmanager"/>
```
Comment (jamezp) [inline feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/main/module.xml]: What makes this required?

**Review by bstansberry — APPROVED** [review]

**Comment by jamezp** [comment]
I'm going to merge this, but I'd like to know more details on why the `org.jboss.logmanager` is required. If the `java.util.logging.manager` system property is expected to be set this makes the logging subsystem no longer optional if the messaging subsystem is used.

File: feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/journal/main/module.xml [hunk 2]
```diff
@@ -27,18 +27,14 @@
...
 
     <dependencies>
         <!-- required for ARTEMIS-298 -->
         <module name="com.google.guava"/>
         <module name="javax.api"/>
         <module name="org.apache.commons.beanutils" />
-        <!-- the journal module depends on org.apache.activemq.artemis so that
-             Artemis's ClassLoadingUtil (from artemis-commons.jar) can load classes
-              from the org.apache.activemq.artemis module) -->
-        <module name="org.apache.activemq.artemis" />
```
Comment (bstansberry) [inline feature-pack/src/main/resources/modules/system/layers/base/org/apache/activemq/artemis/journal/main/module.xml]: @jfdenise ^^^ is merged :)

**Pull request merged**